Update open-rdp.md (#363)
aquasecurity/cloud-security-remediation-guides: en/aws/ec2/open-rdp.md

4. Select the "EC2 Security Group" that needs to be verified. </br> <img src="/resources/aws/ec2/open-rdp/step4.png"/>
5. Scroll down the bottom panel and choose "Inbound" under the "Settings". Verify the value for "Source" column for "RDP" under "Type" for port "3389" and if any rule have value set to "0.0.0.0/0" or "::/0 " then the selected "Security Group" has "TCP" port for "RDP" open to the public.</br> <img src="/resources/aws/ec2/open-rdp/step5.png"/>
6. Repeat steps number 2 - 5 to verify other "Security Groups" in the selected AWS region.</br> 
7. Navigate to "Security Groups" under "Network & Security" and select the "Security Group" that needs to modify to restrict the access of "TCP" port 3389 for "RDP"  to specific ip address. </br> <img src="/resources/aws/ec2/open-rdp/step7.png"/>
8. Scroll down the page and select the "Inbound" and click on the "Edit" button. </br> <img src="/resources/aws/ec2/open-rdp/step8.png"/>
9. In the "Edit inbound rules" tab select either the "MyIP" or "Custom" from the "Source" column.</br> <img src="/resources/aws/ec2/open-rdp/step9.png"/>

> 10. In the "Edit inbound rules" tab select the "MyIP" from the "Source" column to allow "RDP" inbound traffic only from specific IP address, select the Protocol as TCP/UDP or as needed, Action as "Allow/Deny".</br> <img src="/resources/aws/ec2/open-rdp/step10.png"/>

Reviewer: ```suggestion
10. In the "Edit inbound rules" tab select the "MyIP" from the "Source" column to allow "RDP" inbound traffic only from specific IP address, select the required Protocol and Action.</br> <img src="/resources/aws/ec2/open-rdp/step10.png"/>
```
